appveyor.yml

```diff
@@ -14,6 +14,7 @@ environment:
   ARCHIVE_WITH_PDB: TRUE
   GE_TEST_SLEEP_SECONDS_ON_HANG: 0
   GE_TEST_LAUNCH_DEBUGGER_ON_HANG: 0
+  TEST_RUN_COUNT: 1
```

Comment:
When you set it via AppVeyor UI does it override this value?

Reply:
Yes, these variables can be overridden using the AppVeyor project settings.
This makes it difficult to override them for one PR temporarily.
Then we should mark them as default values at least.